server/core/templates/credential_update_set_unixcred_partial.html

```diff
@@ -0,0 +1,64 @@
+<div>
+    <form class="row g-2 pb-3 needs-validation" id="newPasswordForm" novalidate>
+        <input hidden type="text" autocomplete="username" />
+        (% let potentially_invalid_input_class = "" %)
+        (% let potentially_invalid_reinput_class = "" %)
+        (% let pwd_equal = true %)
+
+        (% if let PwdCheckResult::Failure with { pwd_equal, warnings } = check_res %)
+            (% let pwd_equal = pwd_equal.clone() %)
+            (% if !warnings.is_empty() %)
+                (% let potentially_invalid_input_class = "is-invalid" %)
+            (% endif %)
+            (% if pwd_equal %)
+                (% let potentially_invalid_reinput_class = "is-invalid" %)
+            (% endif %)
+        (% endif %)
```

Comment:
This kind of logic should be in the rust-side of the code *please*. There's far too much code sneaking into templates where it's harder to analyse/test, and it's just unnecessary.

Reply:
This was actually @ToxicMushroom not @weijiangan here. It's a copy paste from the password form. I'll address both.